[Master] Added `Extending User Self Registration and Account Confirmation` (#3467)
wso2/docs-is: en/docs/guides/identity-lifecycles/self-registration-workflow.md

    </tr>
    <tr class="even">
    <td>Lock user account on creation</td>
    <td>Enable account lock during self-registration. The account will be unclocked upon confirmation.</td>
    </tr>
    <tr class="odd">
    <td>Manage notifications sending internally</td>
    <td>
    <p>

>     Select to configure the Identity server to send confirmation emails to the user.

Reviewer: ```suggestion
    Enable the Identity server to send confirmation emails to the user.
```

        -   If a user does not consent to any non-mandatory attributes, WSO2 Identity Server will not store those attributes.

7. Click **Finish**.

8. Depending on your requirement, you can either add another new purpose and related user attributes or click **Finish** if you have added all the purposes you want.

9. Click **Update**.

Now you have configured the required self-registration purposes and user attributes for which you require user consent.

Next, you can try out self-registration.

## Try it out

### Use the My Account portal

1. Access the WSO2 Identity Server My Account (`https://<IS_HOST>:<PORT>/myaccount/`).

    ![register-now]({{base_path}}/assets/img/guides/register-now-option.png)

2. Click **Create Account** and then enter the new user's username.

    !!! info "Register Users for a Tenant"
        If you want to self-register to a specific tenant, you need to provide the **Username** in the following format: `<USERNAME>@<TENAND_DOMAIN>`

        For example, if you have a tenant domain as `foo.com`, the username needs to be ` kim@foo.com `

    ![register-users-for-tenant]({{base_path}}/assets/img/guides/register-users-for-tenant.png)

> 3. Fill in the user details, provide consent to share the requested information and then click **Register**.

Reviewer: ```suggestion
3. Fill in the user details, provide consent to share the requested information, and then click **Register**.
```

    - [Guide: Invitation Workflow]({{base_path}}/guides/identity-lifecycles/invitation-workflow) 
    - [Guide: Bulk Import Users]({{base_path}}/guides/identity-lifecycles/bulk-import-users)
    - [Guide: Outbound Provisioning]({{base_path}}/guides/identity-lifecycles/outbound-provisioning) 

>     - [References: Self-Registration verification mechanisms]({{base_path}}/references/extend/user-mgt/self-register-verification)

Author: ```suggestion
    - [Guides: Configure Account Confirmation Methods for Self-Registration]({{base_path}}/references/extend/user-mgt/self-register-verification)
```
